crates/runtime/src/flight.rs

```diff
@@ -167,6 +168,7 @@ impl FlightService for Service {
         &self,
         request: Request<Streaming<FlightData>>,
     ) -> Result<Response<Self::DoPutStream>, Status> {
+        measure_scope!("flight", "operation" => "do_put");
```

Comment:
What does this actually measure? This code returns a stream, so I don't think this is measuring what we think it will measure?

Reply:
You're right, this will complete recording once its dropped out of scope (i.e. after the streams are initialised, but (at least potentially) before the streams are read).